backend/settings.py

```diff
@@ -154,12 +154,26 @@
 # So request works over the proxy
 SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
 
-DB_CACHE_CONFIGS = os.getenv('DB_CACHE_CONFIGS',
-                           {'CACHE_TTL': 600, 'BACKEND': 'django_mysql.cache.MySQLCache',
-                            'LOCATION': 'canvas_app_explorer_cache',
-                            'CACHE_KEY_PREFIX': 'app_explorer',
-                            'CACHE_OPTIONS': {'COMPRESS_MIN_LENGTH': 5000, 'COMPRESS_LEVEL': 6}
-                            })
+DB_CACHE_CONFIGS = os.getenv(
+    'DB_CACHE_CONFIGS',
+    # {
+    #     'CACHE_TTL': 600,
+    #     'BACKEND': 'django_mysql.cache.MySQLCache',
+    #     'LOCATION': 'canvas_app_explorer_cache',
+    #     'CACHE_KEY_PREFIX': 'app_explorer',
+    #     'CACHE_OPTIONS': {'COMPRESS_MIN_LENGTH': 5000, 'COMPRESS_LEVEL': 6}
+    # },
+    {
+        'CACHE_TTL': 600,
+        'BACKEND': 'django_redis.cache.RedisCache',
+        'LOCATION': f"redis://{os.getenv('REDIS_HOST', 'redis')}:6379/1",
+        'CACHE_KEY_PREFIX': 'app_explorer',
+        'CACHE_OPTIONS': {
+            'CLIENT_CLASS': 'django_redis.client.DefaultClient',
+            'PASSWORD': os.getenv('REDIS_PASS')
```

Comment:
Should there be some default password string assigned, in case the env value is missing?

Reply:
By default in the local environment there isn't a password set and it starts up without it. But it's also not exposed to the internet and just internal. I think it would just give an error if it has a password and it's not correct, which would need to be addressed.

Reply:
I updated the documentation to reflect this.